roles/openshift_metrics/defaults/main.yaml

```diff
@@ -59,3 +59,6 @@ openshift_metrics_cassandra_pvc_access: "{{ openshift_hosted_metrics_storage_acc
 openshift_metrics_hawkular_user_write_access: False
 
 openshift_metrics_heapster_allowed_users: system:master-proxy
+
+openshift_metrics_cassandra_enable_prometheus_endpoint: True
```

Comment:
Someone could want to collect metrics just for one or not the other. Or we may want to disable metrics in one to see if that is causing problems.